libcxx/include/__functional/operations.h

```diff
@@ -41,12 +40,12 @@ struct _LIBCPP_TEMPLATE_VIS plus
 };
 _LIBCPP_CTAD_SUPPORTED_FOR_TYPE(plus);
 
-template <class _Tp>
-struct __is_trivial_plus_operation<plus<_Tp>, _Tp, _Tp> : true_type {};
+template <class _Pred>
+struct __desugars_to<plus<_Pred>, plus<_Pred>> : true_type {};
```

Comment:
I think this should be something like

```
template <class _Tp>
struct __desugars_to<plus<_Tp, _Tp>, plus<>> : true_type {};
```

This comment applies below to `std::equal_to` as well.